src/viewer.js

```diff
@@ -1401,7 +1426,7 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
      * @param {Object} [options.ajaxHeaders]
      *      A set of headers to include when making tile AJAX requests.
      *      Note that these headers will be merged over any headers specified in {@link OpenSeadragon.Options}.
-     *      Specifying a falsy value for a header will clear its existing value set at the Viewer level (if any).
+     *      Is this outdated? -> Specifying a falsy value for a header will clear its existing value set at the Viewer level (if any).
```

Comment:
Good point! It certainly doesn't seem to be functional at the moment. Please remove.

Reply:
It comes from [this commit](https://github.com/openseadragon/openseadragon/commit/6fdf81f26676a712966044a3bc7ced2fcdfc0a8f), and the relevant code is now [this](https://github.com/openseadragon/openseadragon/blob/6981cfed0195011ed4191cde9965157388f78e69/src/openseadragon.js#L2431C17-L2437).

Reply:
Thank you for tracking it down! It seems like a potentially useful feature, but I'm guessing it hasn't worked for a while and I don't think anyone has complained. I'm okay with either implementing it or removing the comment.

Reply:
Correct me if I'm wrong, but I think it's functional (the `&& headers[headerName]` condition) in the [current code](https://github.com/openseadragon/openseadragon/blob/6981cfed0195011ed4191cde9965157388f78e69/src/openseadragon.js#L2431C17-L2437).